README.md

```diff
@@ -1 +1,76 @@
-# environment-migration-test
\ No newline at end of file
+# environment-migration-test
+
+# Environment Migration Script User Guide
+
+Required input:
+
+* Source Repo Name
+* Target Repo Name
+* Source Repo Org Name
+* Target Repo Org Name
+* Personal Access Token For Source (repo, users, and org:admin)
+* Personal Access Token For Target (repo, users, and org:admin)
+
+Inputs will be imported as environment variables with the following names:
+
+* SOURCE_REPO
+* TARGET_REPO
+* SOURCE_ORG
+* TARGET_ORG
+* GH_PAT_SOURCE
+* GH_PAT_TARGET
```

Comment:
This information can be consolidated to be more easily consumable

Reply:
I figured that, I will fix

Reply:
done